### PR TITLE
Carousel refine

### PR DESCRIPTION
**Pull request by dprevoznik**
Hello please look over my code. At one point, I have an extra mapping where I use the variable "nextIndex". The purpose of this is a little visual trick to make the carousel look like the data continues to the user. If there is still related products after the subset given, I add one more item to the display that is only slightly in the carousel's view, but doesn't count in the subset shown.

### CODE REVIEW

File: src/components/productDetails/list.jsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ const List = ({
   var [outfitsReviews, setOutfitsReviews] = useState([]);
 
   // Set Local State For Conditionally Rendered Products
-  let initialShownIndices = listName === "Outfit" ? [0, 1, 2] : [0, 1, 2, 3];
+  let initialShownIndices = listName === "Outfit" ? [0, 1, 2, 3] : [0, 1, 2, 3];
   var [shownIndices, setShownIndices] = useState(initialShownIndices);
```
Comment (iwantmyhatback) [inline src/components/productDetails/list.jsx:33]: i am assuming the addition of the ',3' insertion here is what you were referencing for the faux next item tile. very cool

File: src/components/productDetails/list.jsx [hunk 2]
```diff
@@ -112,61 +113,126 @@ const List = ({
...
-            />
+    <>
+      <h1 className="title">{listName} Items</h1>
+      <div className="columns">
+        <div className="column is-narrow is-vertical-centered">
+          <span>
+            {shownIndices[0] !== 0 ? (
+              <DirectionalButton
+                arrowDirection={"left"}
+                icon={"fas fa-chevron-left"}
```
Comment (iwantmyhatback) [inline src/components/productDetails/list.jsx:137]: after seeing all the references to "fas" in your code i finally looked up what that was. i had no idea that bulma has all of these icons preloaded!

**Review by iwantmyhatback — APPROVED** [review]
Everything looks good. Approved

File: src/components/productDetails/list.jsx (hunk 2 above)
Comment (dprevoznik) [reply src/components/productDetails/list.jsx:137]: It doesn't actually. If you go look at our index.html file we are also loading font awesome.

**Pull request merged**